coalib/coala.py

```diff
@@ -59,26 +60,49 @@ def main(debug=False):
 
         if args.show_bears:
             from coalib.settings.ConfigurationGathering import (
-                get_filtered_bears)
-
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, log_printer)
-
+                get_all_bears)
+            filtered_bears = get_all_bears(log_printer)
+            if args.filter_by_language:
+                # filter_by_language option should be deprecated soon.
+                # Since it is already covered by filter_by option.
+                log_printer.warn(
+                    "'--filter-by-language ...' option is deprecated. "
+                    "Use '--filter-by language ...' option instead")
+                if args.filter_by is None:
+                    args.filter_by = []
+                filter_by_x = ['language']
+                filter_by_x.extend(args.filter_by_language)
+                args.filter_by.append(filter_by_x)
```

Comment:
Have to keep `append` instead of `+=` as it is args and being used in the else part also. 
Rest of the suggestion is applied.